Description: Use /etc/default/keyboard instead of /etc/vconsole.conf
Forwarded: not-needed

--- a/src/libply-splash-core/ply-terminal.c
+++ b/src/libply-splash-core/ply-terminal.c
@@ -131,9 +131,9 @@ ply_terminal_parse_keymap_conf (ply_term
         if (keymap)
                 return keymap;
 
-        vconsole_conf = ply_key_file_new ("/etc/vconsole.conf");
+        vconsole_conf = ply_key_file_new ("/etc/default/keyboard");
         if (ply_key_file_load_groupless_file (vconsole_conf))
-                keymap = ply_key_file_get_value (vconsole_conf, NULL, "KEYMAP");
+                keymap = ply_key_file_get_value (vconsole_conf, NULL, "XKBLAYOUT");
         ply_key_file_free (vconsole_conf);
 
         /* The keymap name in vconsole.conf might be quoted, strip these */
